### ISSUE TITLE
Files getting 'damaged'

### ISSUE THREAD

**Issue by Kitanga** [Sevin777/Ace.Tern#39]
I don't even know what to call this, but here this is what I tend to get with your app http://imgur.com/4zYDNOP I thought it was because of my laptop suddenly shutting down but today I closed it before my laptop went off and still this happened. The image you see was my home page for a site I was working on.

**Comment by Sevin777** [comment]
I'm assuming you're talking about Caret-T. I doubt that the app is turning your files into a chinese-like language (unless it was already like that)? However, there is a [known bug](https://code.google.com/p/chromium/issues/detail?id=474183) that affects all Chrome Apps that only occurs when dealing with rare special characters in a file.

**Comment by Kitanga** [comment]
Yes I was talking about caret-t. Here's what it looks like in the editor.
